flights/berks_ceh_conditions_flight.yaml

```diff
@@ -82,32 +94,79 @@ entityDefinitions:
         - !<transforms.ReplaceTransform>
           target: ["5", "6", "7", "8", "9", "10"]
           goal: ["Physical disability", "Developmental disability", "Chronic health condition", "HIV/AIDS", "Mental health problem", "Substance abuse"]
-      ol.description:
-        type: "ol.description"
+      ol.clinicalstatus: 
+        type: "ol.clinicalstatus"
         transforms: 
         - !<transforms.BooleanContainsTransform>
           column: "DisabilityType"
           string: "10"
           transformsIfTrue:
-          - !<transforms.ColumnTransform>
             column: "DisabilityResponse"
```

Comment:
please take this one out and put it in the entity type ol.substance, then associate it with a person. Similar to how you did pregnancy.